apps/web/utils/rule/consts.ts

```diff
@@ -7,4 +7,11 @@ export const RuleName = {
   Receipt: "Receipt",
   Notification: "Notification",
   ColdEmail: "Cold Email",
+  ToReplyDigest: "To Reply Digest",
+  NewsletterDigest: "Newsletter Digest",
+  MarketingDigest: "Marketing Digest",
+  CalendarDigest: "Calendar Digest",
+  ReceiptDigest: "Receipt Digest",
+  NotificationDigest: "Notification Digest",
+  ColdEmailDigest: "Cold Email Digest",
```

Comment:
remove these. we don't need special names for these rules. newsletter rule can still be called newsletter